### DISCUSSION TITLE
Open GL Es 3.1

### DISCUSSION THREAD

**Discussion by JwSh-1** [skyline-emu/skyline#195] [Ideas]
Skyline has OpenGL support?

In Android 10 and 11, the emulators have more fps with Open GL, than with vulkan. Would it be feasible to implement it in Skyline?

**Comment by bylaws** [comment]
Skyline will never support opengl in our lifetime

**Reply by JwSh-1** [reply]
Is there a reason why Opengl will not be implemented?

**Reply by bylaws** [reply]
Why would we when vulkan exists
